src/redux/actionTypes/projects.js

```diff
@@ -0,0 +1,18 @@
+const PROJECTS = 'projects';
+
+/**
+ * Save project.
+ */
+const PROJECTS_CREATE = `${PROJECTS}/create`;
+
+/**
+ * Update project.
+ */
+const PROJECTS_UPDATE = `${PROJECTS}/update`;
+
+/**
+ * Update project.
+ */
+const PROJECTS_SET_ACTIVE = `${PROJECTS}/setActive`;
```

Comment:
why is there a second action for updating a project?

Reply:
This one is for setting the active project

Reply:
can you update the comment on top of it in this case?